XiEditor/AppDelegate.swift

```diff
@@ -83,7 +83,7 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
     var documentController: XiDocumentController!
 
     // This is set to 'InconsolataGo' in the user preferences; this value is a fallback.
```

Comment:
This comment is out of date. It's also not totally clear that we need a fallback font now, but we may be relying on it in certain places (like setting up the initial drawing metrics,  below)

Reply:
(to clarify: I think it's fine to leave the fallback)

Reply:
I made the comment change to clarify Inconsolata being bundled now.